ERCS/erc-XXXX.md

```diff
@@ -77,134 +81,166 @@ Example TWIT manifest at <https://my-crypto-dapp.org/twit.json>:
 
 ```json
 {
- "publicKeys": [
-    {"id": "1", "alg":"ECDSA", "publicKey": "0xaf34..."},
-    {"id": "2", "alg":"RSA-PSS", "publicKey": "0x98ab..."}
+  "publicKeys": [
+    { "id": "1", "alg": "ECDSA", "publicKey": "0xaf34..." },
+    { "id": "2", "alg": "RSA-PSS", "publicKey": "0x98ab..." }
   ]
 }
 ```
 
+Dapps SHOULD only rely on algorithms available via [SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto), since they are present in every browser.
+
 #### Manifest schema
 
-We are proposing a very simple schema that will allow for future security improvements if needed.
+We propose a simple and extensible schema:
 
 ```json
 {
- "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "title": "TWIT manifest",
-    "type": "object",
-    "properties":{
-        "publicKeys": {
-         "type": "array",
-            "items":{
-             "type": "object",
-                "properties": {
-                  "id": { "type": "string"},
-                  "alg": { "type": "string"},
-                  "publicKey": { "type": "string"}
-                }
-            }
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "TWIT manifest",
+  "type": "object",
+  "properties": {
+    "publicKeys": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": { "type": "string" },
+          "alg": { "type": "string" },
+          "publicKey": { "type": "string" }
         }
+      }
     }
+  }
 }
 ```
 
-#### Example implementation
+#### RPC method
 
-We propose to rely on algorithms supported by [SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto) as they are present in every browsers.
+The parameters of `wallet_signedRequest` are specified by this TypeScript interface:
 
-We recommend adding new method `secureRequest` to EIP-1193 as it would provide a simple, yet extensive, implementation for secure calls. In addition, it provides
-dApp maintainers/devloppers with an explicit way to ask wallets to verify the integrity of the payload, as opposed to an implicit implementations relying on bit
-packing of the arguments.
```

Comment:
should not we add somthing along these lines somewhere ?

Reply:
Added to the end of `Motivation` in [7e72a2e](https://github.com/uni-guillaume/ERCs/pull/2/commits/7e72a2eb1cac851a12f74e4d77f9eb90a598a2c3)